lib/liftoff/xcodeproj_helper.rb

```diff
@@ -40,9 +40,19 @@ def set_indentation_level(level, use_tabs)
     def add_script_phases(scripts)
       if scripts
         scripts.each do |script|
-          key, value = script.first
-          puts "Adding shell script build phase '#{value}'"
-          add_shell_script_build_phase(file_manager.template_contents(key), value)
+
+          # -1 means last as index
+          if script.length > 1
```

Comment:
Instead of supporting multiple formats, maybe we should just switch to this format and make the `index` key optional. That way, we don't need this conditional.